app/controllers/questions_controller.rb

```diff
@@ -16,11 +18,13 @@ def edit
   end
 
   def create
-    @question = Question.create(question_params)
+    @question = Question.create question_params.merge(user: current_user)
 
     if @question.save
+      flash[:notice] = 'Your question successfully created.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.